src/Security/UserClassBuilder.php

```diff
@@ -232,7 +232,7 @@ private function addGetSalt(ClassSourceManipulator $manipulator, UserClassConfig
         if ($userClassConfig->hasPassword()) {
             $builder->addStmt(
                 $manipulator->createMethodLevelCommentNode(
-                    'not needed when using the "bcrypt" algorithm in security.yaml'
+                    'not needed when using the "bcrypt/sodium" algorithm in security.yaml'
```

Comment:
```suggestion
                    'not needed when using a modern encoder algorithm in security.yaml'
```

@BatsaxIV I think it's safe to go with your suggestion. I'll take the heat if @weaverryan disagrees :P